packit_service/worker/handlers/github_handlers.py

```diff
@@ -569,7 +570,7 @@ def run(self) -> HandlerResults:
 
         api = PackitAPI(
             config=self.config,
-            package_config=self.package_config,
+            package_config=self.job_config,
```

Comment:
This was a little bit confusing when I read it for the first time (maybe we could rename the attribute later in packit)

Reply:
agreed, `common_config` could be a better name

Reply:
Or, use `service_config` for service config and `config` for the general one. (We discussed the expliciteness of `service_config` in her last PR..)